packages/react/src/patterns/blocks/ContentGroupPictograms/__stories__/ContentGroupPictograms.stories.js

```diff
@@ -100,6 +100,11 @@ export default {
             'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
             groupId
           ),
+          copy: text(
+            'Copy (optional)',
```

Comment:
@guilhermelMoraes since you're updating the story, can you update the knob labels for this story to include the actual prop name in parenthesis?

Reply:
I'm sorry, I didn't get what you mean. Perhaps set it to: `Copy (props.copy)(optional)`?

Reply:
Right, well something like: https://ibmdotcom-react-canary.mybluemix.net/?path=/story/components-masthead--default

If you can add for the other props in this story too would be great!